example/README.md

```diff
@@ -29,7 +29,7 @@ brew install tfenv
 In order to prevent state corruption and other issues, you **should only install and use the version of Terraform specified in the `example/.tfversion` file**:
 
 ```shell
-tfenv install $(cat example/.tfversion)
+env TFENV_ARCH=amd64 tfenv install $(cat example/.tfversion)
```

Comment:
I left it as explicit because it'll work as expected an Intel Mac or ARM-equipped. If you're on an Intel Mac, that's the config it would use implicitly anyway, so no harm in specifying it.

Reply:
Sorry, just flagged that and noted the discussion, that seems fine given this is Mac-centric, ignore the other comment.